go/vt/vtgate/planbuilder/testdata/foreignkey_cases.json

```diff
@@ -1364,7 +1453,7 @@
                   "Sharded": false
                 },
                 "TargetTabletType": "PRIMARY",
-                "Query": "update u_multicol_tbl2 set cola = null, colb = null where (cola, colb) in ::fkc_vals and (cola, colb) not in ((1, 2))",
+                "Query": "update u_multicol_tbl2 set cola = null, colb = null where (cola, colb) in ::fkc_vals and (1 is null or (cola, colb) not in ((1, 2)))",
```

Comment:
This change will go away when https://github.com/vitessio/vitess/pull/14059 is merged.

Reply:
The PR has been merged and the expectations changed back.